packages/host/app/components/operator-mode/container.gts

```diff
@@ -475,6 +471,29 @@ export default class OperatorModeContainer extends Component<Signature> {
         format: 'isolated',
         stackIndex: this.operatorModeStateService.state.stacks.length,
       });
+    } else {
+      // In case, that the search was accessed directly without clicking right and left buttons,
+      // the rightmost stack will be REPLACED by the selection
+      let numberOfStacks = this.operatorModeStateService.numberOfStacks();
+      if (numberOfStacks > 0) {
+        //there will always be 1 stack
+        let stack = this.operatorModeStateService.rightMostStack();
+        if (stack) {
+          let bottomMostItem = stack[0];
+          if (bottomMostItem) {
```

Comment:
couldn't you use your new `stackIsEmpty()` here? then line 491 is just `stack[0]`. that way there are less variable declarations